plugins/setting-resources/src/components/Settings.svelte

```diff
@@ -39,6 +39,9 @@
     setMetadataLocalStorage(login.metadata.LoginEmail, null)
     navigate({ path: [login.component.LoginApp] })
   }
+  function selectWorkspace (): void {
+    navigate({ path: [login.component.LoginApp, 'selectWorkspace'] })
```

Comment:
You can navigate to LoginApp, 'selectWorkspace' must be selected automatically if token exist